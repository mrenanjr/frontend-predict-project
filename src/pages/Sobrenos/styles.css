.container-principal .img {
    height: 200px;
    background-color: #C0C0C0;
    text-align: center;
}

.container-principal .img  img {
    height: 100%;
    width: 100%;
}

.container-principal .sobrenos-text {
    color: white;
    font-size: 16px;
}