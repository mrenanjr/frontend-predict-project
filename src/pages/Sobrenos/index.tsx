import React from 'react';
import './styles.css'
import Footer from '../../components/Footer';
import Header from '../../components/Header';

import Img from '../../assets/insert_photo-24px.svg';

const Inicio = () => {
    return (
        <>
            <Header />
            <main className="container-principal">
                <div className="row" style={{ height: '100%' }}>
                    <div className="col-2 img">
                        <img src={Img} alt="Imagem"/>
                    </div>
                    <div className="col-9">
                        <p className="sobrenos-text">
                                Observações e análises de fenômenos como a retenção e evasão no ensino superior
                            público é de suma importância visto que, compromete a taxa de de conclusão nos cursos de Bacharelado e Licenciatura, gera ociosidade de recursos humanos e materiais e não retorna os
                            investimentos aplicados pelos recursos públicos à própria sociedade. Assim os impactos
                            negativos que a retenção e a evasão causam na sociedade e na própria UFG são sensíveis e
                            importantes de serem investigados e estudados.
                        </p>
                    </div>
                    <div className="col-12">
                        <p className="sobrenos-text">
                                Posto que a sociedade é forçada a custear
                            estes estudantes e que eles não irão dar um retorno ao investimento aplicado em sua
                            formação, dentro do período previsto se prolongando no curso (retenção) ou até mesmo
                            inviabilizando seu retorno como um profissional graduado (evasão) regressando, assim, os
                            recursos investidos durante sua permanência na universidade. Estes estudantes ocupam uma
                            vaga pública que poderia ter sido mais bem aproveitada.
                        </p>
                        <p className="sobrenos-text">
                                Ainda, de acordo com o site “somospar” a taxa de evasão escolar no Brasil é a terceira
                            maior do mundo com média de 24,1% na lista dos 100 países com melhor IDH. Existem
                            algumas causas listadas pelo site “politize.com.br” que são: Acesso limitado; Necessidade
                            especial; Gravidez e Maternidade; Atividades ilegais; Mercado de trabalho; Pobreza; Violência;
                            Deficit de aprendizagem; Qualidade da educação e etc.
                        </p>
                        <p className="sobrenos-text">
                                O projeto discorre da procura em diminuir a quantidade de retenção e evasão de
                            estudantes dos cursos da Universidade Federal de Goiás (UFG), diminuindo sua taxa de
                            retenção e evasão em pelo menos 50% em 5 anos comparado ao total.
                        </p>
                    </div>
                </div>
            </main>
            <Footer />
        </>
    );
}

export default Inicio;
