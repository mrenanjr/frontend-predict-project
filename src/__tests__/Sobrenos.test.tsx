import * as React from 'react';
import { shallow } from 'enzyme';
import Sobrenos from '../pages/Sobrenos';

test('Sobrenos loads corretly', () => {
  const wrapper = shallow(<Sobrenos />);

  expect(wrapper).toMatchSnapshot();
});