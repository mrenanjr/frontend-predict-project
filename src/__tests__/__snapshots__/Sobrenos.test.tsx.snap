// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sobrenos loads corretly 1`] = `
<Fragment>
  <Header />
  <main
    className="container-principal"
  >
    <div
      className="row justify-content-center"
      style={
        Object {
          "height": "100%",
        }
      }
    >
      <div
        className="col-3 img"
      >
        <img
          alt="Imagem"
          src="test-file-stub"
        />
      </div>
      <div
        className="col-12"
      >
        <p
          className="sobrenos-text"
        >
                   O Sistema de Análise Preditiva – SISAP é o produto final funcional proveniente da Especialização em Design de Sistemas e Soluções de Business Inteligence – DSSBI e da Especialização em Governança e Gestão de Sistemas e Tecnologias da Informação – EspGTI do Instituto de Informática da Universidade Federal de Goiás.
          <br />
          <br />
                   O SISAP trata-se de um modelo preditivo com uma interface amigável para retorno dos resultados obtidos pelo algoritmo. O algoritmo foi criado e testado com dados extraídos do histórico acadêmico de alguns cursos em uma Instituição de Ensino Superior (IES) no período compreendido entre 2008 a 2018. De posse destes dados, foram avaliados os principais indicadores que poderiam ser apontados como fatores influentes para o problema da evasão através de um algoritmo que utiliza “
          <i>
            machine learning
          </i>
          ” ou aprendizado de máquina.
          <br />
          <br />
                   O algoritmo foi elaborado a partir de um estudo longitudinal, resultando em um modelo de classificação preditiva uma acurácia que supera os 70% (setenta por cento).
          <br />
          <br />
                   Assim o SISAP é um “
          <i>
            dashboard
          </i>
          ” onde as métricas consolidadas são organizadas de modo inteligível, com linguagem de fácil acesso aos usuários, contribuindo assim para gestão acadêmica, possibilitando a tomada de ações tendo em vista discentes apontados pelo algoritmo com alta probabilidade de evasão.
          <br />
          <br />
              O sistema encontra-se em sua Versão 1.0 criada em agosto/2020
          <br />
          <br />
              Estudantes do Projeto:
          <br />
          <br />
          <b>
                Especialização em Design de Sistemas e Soluções de Business Inteligence – DSSBI:
          </b>
          <br />
              Anderson de Oliveira Costa
          <br />
              Breno Rodrigo da Silva
          <br />
              Manoel Renan Oliveira Júnior
          <br />
          <br />
          <b>
                Especialização em Governança e Gestão de Sistemas e Tecnologias da Informação – EspGTI:
          </b>
          <br />
              Daniel Chaffe Stone
          <br />
              Diogo Sebastião Fernandes Guimarães
          <br />
              Franceles Bezerra de Oliveira
          <br />
              Joelson Coelho Costa
          <br />
              Leila Aparecida Santos Motta Cunha
          <br />
              Matheus Silva Santos
          <br />
        </p>
      </div>
    </div>
  </main>
  <Footer />
</Fragment>
`;
